admin/kernel-masterdata-service/src/main/java/io/mosip/kernel/masterdata/service/impl/ApplicationConfigServiceImpl.java

```diff
@@ -30,6 +31,9 @@ public class ApplicationConfigServiceImpl implements ApplicationConfigService {
 		
 	@Value("${mosip.admin.ui.configs}")
 	private String uiConfigs;
+	
+	@Value("${mosip.right_to_left.orientation:#{null}}")
```

Comment:
If it is only ui related configuration, then it will be better the name the key "mosip.admin.ui.*"

Reply:
removed the same, as it is no longer needed